tests/cdi/cdi_test.go

```diff
@@ -0,0 +1,53 @@
+package cdi_test
+
+import (
+	"os"
+	"flag"
+
+	. "github.com/onsi/ginkgo"
+	"kubevirt.io/kubevirt-ansible/tests"
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+// template parameters
+const (
+	pvcEPHTTPNOAUTHURL = "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img"
+	pvcName            = "golden-pvc"
+	vmName             = "test-vm"
+	vmAPIVersion       = "kubevirt.io/v1alpha2"
+	rawPVCFilePath     = "tests/manifests/golden-pvc.yml"
+	rawVMFilePath      = "tests/manifests/test-vm.yml"
+	sepP               = "-p"
+)
+
+var _ = Describe("Importing and starting a VM using CDI", func() {
+	flag.Parse()
+	ktests.SkipIfNoOc()
+	dstPVCFilePath := "/tmp/test-pvc"
+	dstVMFilePath := "/tmp/test-vm"
+
+	url, ok := os.LookupEnv("STREAM_IMAGE_URL")
+	if !ok {
+		url = pvcEPHTTPNOAUTHURL
+	}
+
+	Context("with valid image url will succeed", func() {
+		It("should create the PVC successfully", func() {
+			tests.ProcessTemplateWithParameters(rawPVCFilePath, dstPVCFilePath, sepP, "PVC_NAME="+pvcName, sepP, "EP_URL="+url)
+			tests.CreateResourceWithFilePath(pvcName, "persistentvolumeclaim", dstPVCFilePath+".json")
+		})
+
+		Specify("the PVC should become bound", func() {
+			tests.WaitUntilResourceReady(pvcName, "pvc", "-o=jsonpath='{.metadata.annotations}'", "pv.kubernetes.io/bind-completed:yes pv.kubernetes.io/bound-by-controller:yes")
+		})
+
```

Comment:
I think you want to add a step to check that the import has started.  You could use the WaitUntilResourceReady call to check for the proper pvc annotation.  Otherwise you risk the VM start flow racing with the CDI import pod creation.